pages/installation/os/centos.rst

```diff
@@ -85,6 +85,9 @@ The last step is to enable Graylog during the operating system's startup::
   $ sudo systemctl start graylog-server.service
 
 
+.. note:: We higly recommend that you proxy the Graylog Webinterface and API with NGINX/Apache like described in: :ref:`configuring_webif_nginx`
```

Comment:
IMHO this Step-by-step is used by people that are not very skilled on linux and for them it is easier to run HTTPS from NGINX/Apache.